vignette/Validation.Rmd

```diff
@@ -0,0 +1,155 @@
+---
+title: "Comparison"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Comparison}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>", 
+  fig.path = "man/figures/README-",
+  fig.width = 9, 
+  fig.height = 9
+)
+```
+
+```{r setup}
+library(hypatia)
+library(ggplot2)
+library(tidyverse)
+```
+
+```{r hypatia}
+
+# Create our population
+countryname = "Antigua and Barbuda"
+
+pop <- hypatia:::get_population(countryname)
+
+# Scale it for speed
+pop$n <- round(pop$n/10)
+
+# Create our equivalent parameters
+time_period <- 200
+
+parameters <- get_parameters(
+  population = pop$n,
+  contact_matrix_set = squire::get_mixing_matrix(iso3c = "ATG"),
+  R0 = 2,
+  time_period = time_period,
+  tt_contact_matrix = 0
+)
+
+# run our simulation - default timesteps = 200
+df <- hypatia:::run_simulation(pop, parameters, max_age = 100)
+
+numruns <- length(df)
+numdatapoints <- length(df[[1]][[1]]) - 1
+subtitle <- paste(
+  "Simulation for 1 run and 200 data points per run")
+
+df <- dplyr::bind_rows(df, .id = "group")
+
+statesnamevector <- c("human_S_count",
+                           "human_E_count",
+                           "human_IMild_count",
+                           "human_ICase_count",
+                           "human_IOxGetLive_count",
+                           "human_IOxGetDie_count",
+                           "human_IOxNotGetLive_count",
+                           "human_IOxNotGetDie_count",
+                           "human_IMVGetLive_count",
+                           "human_IMVGetDie_count",
+                           "human_IMVNotGetLive_count",
+                           "human_IMVNotGetDie_count",
+                           "human_IRec_count",
+                           "human_R_count",
+                           "human_D_count")
+
+  # Convert to long format
+df <- tidyr::pivot_longer(tibble::as_tibble(df), all_of(statesnamevector))
+
+strname <- "Hypatia Model Simulation"
+subtitle <- paste(
+    "Simulation for 1 run and 1000 data points")
+
+time <- seq(1,200,1)
+
+timsestep <- df$timestep
+value <- df$value
+name <- df$name
+group <- df$group
+
+ggplot2::ggplot(
```

Comment:
Can you get rstudio to reindent this please? it should not line up against the left edge after the first `+`